templates/openstack/cis.html

```diff
@@ -0,0 +1,145 @@
+              alt="",
+              width="22",
+              height="16",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>
+          <td>
+            {{ image (
+              url="https://assets.ubuntu.com/v1/87f100d4-tick.svg",
+              alt="",
+              width="22",
+              height="16",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <p>
+      <a href="/support#get-in-touch" class="p-button--positive">Access the Ubuntu CIS benchmark tooling</a>
+      <a href="https://security-certs.docs.ubuntu.com/en/cis" class="p-button--neutral p-link--external">Learn more about our tools</a>
+    </p>
+    <h3>What are the CIS Controls?</h3>
+    <p><a class="p-link--external" href="https://www.cisecurity.org/controls/">CIS controls</a>, is a framework of security best practices, that harness the collective experience of the CIS subject matter experts from actual attacks and effective defenses. CIS controls are referenced by International and National frameworks such ETSI’s critical security controls, NIST Cybersecurity framework, and others.</p>
+    <h3>How do benchmarks relate with CIS Controls?</h3>
+    <p>The benchmarks map to <a class="p-link--external" href="https://www.cisecurity.org/controls/">CIS controls</a> and are designed to  additionally reduce the system’s attack surface to mitigate the most common attacks. For that reason, they are considered a necessary complement in the implementation of a cybersecurity framework, and are the most widely accepted Industry benchmark to harden a system today.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6 u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
+        alt="",
+        width="386",
+        height="220",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h3>Configure and apply CIS hardening rules in minutes</h3>
+      <p>CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides CIS automation tooling available to its Ubuntu Advantage customers and Ubuntu Pro users on the public cloud.</p>
+      <p>The compliance tooling has two objectives: it lets our customers harden their Ubuntu systems effortlessly and then quickly audit those systems against the published CIS Ubuntu benchmarks. The SCAP content for audit tooling that scans the system for compliance is CIS certified.</p>
+      <a href="https://www.youtube.com/watch?v=wyEX0eyoK88" class="p-button--neutral p-link--external">Watch the video</a>
+    </div>
+  </div>
+</section>
+
+
```

Comment:
copy the marketo info from the bottom of this [page](https://github.com/canonical-web-and-design/ubuntu.com/pull/9852/files)